lookout/style/format/code_generator.py

```diff
@@ -85,34 +85,19 @@ def generate_new_line(self, line_vnodes: List[VirtualNode]) -> str:
                             exists. The final line break of the line should not be included.
         :return: Code line.
         """
-        if not line_vnodes:
-            return ""
-
-        first_y = line_vnodes[0].y
-
         generated = self.generate(line_vnodes)
-        if line_vnodes[0].y is None:
-            # we add " " because we want to count probably empty last line
-            lines_no = len((line_vnodes[0].value + " ").splitlines())
-            return "".join(generated.splitlines()[lines_no:])
-
-        if first_y is not None and self.NEWLINE_INDEX not in first_y:
-            return generated
-
-        # First line is always removed because it is an end line from the previous line.
-        generated_lines = generated.splitlines(keepends=True)[1:]
-        if not generated_lines:
-            # Usually it means that you replace several newlines with only one.
+        if generated == "":
             return ""
-        for i, line in enumerate(generated_lines):  # noqa B007
-            if line.splitlines()[0]:
-                # Line is not empty
-                break
-        lines_num = 0
-        if line_vnodes and hasattr(line_vnodes[0], "y_old") and self.NEWLINE_INDEX in first_y:
-            lines_num = line_vnodes[0].y.count(self.NEWLINE_INDEX) - \
-                        line_vnodes[0].y_old.count(self.NEWLINE_INDEX)
-        return "".join(generated_lines[i - max(0, lines_num):])
+
+        generated = generated.splitlines(keepends=True)
+        lines_old = sum(getattr(vnode, "y_old", vnode.y).count(NEWLINE_INDEX)
+                        for vnode in line_vnodes if vnode.y is not None)
+        lines_new = sum(vnode.y.count(NEWLINE_INDEX)
+                        for vnode in line_vnodes if vnode.y is not None)
+        # We always sure that the last line is related to requested line so we count from the end
```

Comment:
```suggestion
        # the last line always relates to the requested line so we count from the end
```

What is "requested line"? Not clear